src/index.js

```diff
@@ -826,7 +826,7 @@ follow these steps, YOUR APP MAY BREAK.`);
     const clientPool =
         new ClientPool(MAX_CONCURRENT_REQUESTS_PER_CLIENT, () => {
           const gapicClient =
-              module.exports.v1beta1(this._initalizationOptions);
+              module.exports.v1beta1(this._initalizationSettings);
```

Comment:
What can we do to avoid something like this happening again? Can we somehow cover this in unit tests? Or how about an assertion here:

```
if (typeof this._initializationSettings === 'undefined') {
   throw new Error();
}
```

Reply:
1) Use TypeScript everywhere.
2) Run the system tests without application default credentials in the background. I am going to see how we can add that to CI.